.github/workflows/generated-files.yaml

```diff
@@ -7,7 +7,7 @@ on:
   push:
     branches: [ main ]
   pull_request:
-    branches: [ '**' ]
+    branches: [ dextest ]
```

Comment:
This (and similar changes in workflows) will be removed.  There are added just to avoid triggering tests when not needed